internal/api/genapi/templates.go

```diff
@@ -68,6 +69,7 @@ func fillTemplates() {
 			TypeOnCreate:        in.typeOnCreate,
 			CreateResponseTypes: in.createResponseTypes,
 			RecursiveListing:    in.recursiveListing,
+			SubAction:           in.subAction,
```

Comment:
Are we sure we want to approach sub actions this way (through setting subactions as a string)?  I personally would take the approach of making the pathing changes in this PR be transparent to the API to buy us some time to think about and discuss the best way to support sub actions once that becomes necessary in the API (maybe when we add OIDC).

Reply:
I'm happy to have a conversation about it, but given we already know what OIDC will look like, I think we can have that conversation with the team now, rather than later.

Reply:
Per discussion, removed the API changes